article/Related-work/related-work.tex

```diff
@@ -1 +1,35 @@
+Collaborative filtering(CF) is a widely used technique in modern recommender systems \cite{lightgcn}.
+There are two main disciplines of CF which are the latent factor approach and the neighborhood approach \cite{SVD}.
+Neighborhood approaches compute the realtionship between items or alternatively between users.
+This will essentially transform users into sets of items which can then be compared.
+Latent factor methods represent users and items as parameterized vectors and learn these parameters by reconstructing historical user-item interaction data.
+One of the earliest examples of this is Matrix factorization(MF) which maps users and items in a joint latent factor space with the same dimensionality and then models the user-item interactions as innerproducts in that space \cite{Matrix-factorization-techniques}.
+These representations of users and items are also commonly called embeddings.
+MF then uses the dot product of these to predict an user-item interaction.
+These methods do not require any domain knowledge because they only represent the users and items by their ID without any additional information.
+Newer methods also use neural network to improve this model while still using the older format of embeddings. 
+Examples of this are NCF \cite{NCF} which replaces the inner product with a deep neural network and LRML\cite{LRML} which uses neural networks to improve the learned embeddings.
+SVD++ \cite{SVD} combines the neighborhood and the latent factor approach while also taking implicit feedback into account when calculating the predictions.
+
+\subsection{Graph based recommenders}
+Another reasearch area that is relevant to our paper is Graph based recommenders.
+T
+Burde være muligt at finde alle de kilder man har lyst til i den her. 
+se side 10: https://arxiv.org/pdf/2011.02260.pdf
+
+\subsection{Extensions to LightGCN}
+There have been multiple extensions to LightGCN and work created inspired by LightGCN \cite{LGC-ACF,IMP-GCN,SGNN,BiTGCF}.
+One of them is LightGCN based Aspect-level Collaborative Filtering (LGC-ACF), which utilizes LightGCN and adds side information \cite{LGC-ACF}.
+The input to this model is a user-item graph and a user-side information graph. 
+These two graphs independently use the propagation method from LightGCN  (as seen on \autoref{eq:lightgcn-propagation}).
+For layer combination, LGC-ACF uses means as their aggregation function when combining the user embeddings created from the user-item graph and the user embeddings from the user-side information graph.
+This is also done with the item embedding \cite{LGC-ACF}.
+Interest-aware Message-Passing GCN (IMP-GCN) is an alternative extension to LightGCN \cite{IMP-GCN}. 
+It alleviates the problem of GCNs being over-smoothed by creating clusters within the graphs so that not all nodes will end up being connected.
+The result of this method is that it can utilize a higher number of convolutions with good performance compared to LightGCN \cite{IMP-GCN}. 
```

Comment:
```suggestion
The result of this method is that it can utilize a higher number of convolutions with good performance compared to LightGCN \cite{IMP-GCN}. 
Both of these methods could probably benefit from using a different layer combination method than just using means.
```